src/sentry/conf/server.py

```diff
@@ -867,6 +867,12 @@ def create_partitioned_queues(name):
     'projects:sample-events': True,
     # Enable functionality to trigger service hooks upon event ingestion.
     'projects:servicehooks': False,
+    # Use Kafka (instead of Celery) for preprocess task messages.
+    'projects:kafka-preprocess': False,
+    # Use Kafka (instead of Celery) for process task messages.
+    'projects:kafka-process': False,
+    # Use Kafka (instead of Celery) for save_event task messages.
+    'projects:kafka-save': False,
```

Comment:
I wonder if there's any value to splitting these out or whether it should just be one option like `projects:kafka-ingest-pipeline` or whatever.

Reply:
If you have any permutation of these on, do the others need to also be on? If they need to work as an atomic unit, just make them one feature. I'm not sure if there's any reason you'd use one without the others?

Reply:
None of them require the others, any any stage in the chain it's OK to decide to use Celery instead. But yeah, it doesn't seem useful to me to test different anything more than "all Kafka" since that's where we're headed.